ibis/backends/tests/test_array.py

```diff
@@ -525,6 +631,11 @@ def test_array_remove(backend, con):
     raises=(AssertionError, GoogleBadRequest),
     reason="bigquery doesn't support null elements in arrays",
 )
+@pytest.mark.notimpl(
+    ["flink"],
+    raises=NotImplementedError,
+    reason="Not implemented `from_ibis(cls, dtype: DataType) -> T`",
```

Comment:
No need to restate the error message here.

Reply:
Thanks, modified the `reason`.